client/packages/system/src/Patient/Components/PatientSearchModal/PatientSearchModal.tsx

```diff
@@ -36,6 +39,11 @@ const PatientSearchComponent: FC<PatientSearchModalProps> = ({
     onClose();
   };
 
+  function handlePatientModalClick() {
+    handleClose();
+    openPatientModal();
+  }
```

Comment:
We generally use Arrow function definitions, probably best to stay consistent. I think this is almost small enough to just define it inline with the props, but up to you.